cmd/create/cluster/cmd.go

```diff
@@ -2100,37 +2106,6 @@ func validateExpiration() (expiration time.Time, err error) {
 	return
 }
 
-const (
```

Comment:
I've just refactored lines **2103-2133** to `pkg/aws/helpers.go` to call the function in `pkg/interactive/validation.go`.